Sources/docc/DocCDocumentation.docc/documenting-a-swift-framework-or-package.md

```diff
@@ -71,14 +71,16 @@ step-by-step, interactive content. For more information, see
 > Important: To use a documentation catalog in a Swift package, make sure the 
 manifest's Swift tools version is set to `5.5` or later. 
 
-## Building, publishing, and previewing documentation
+## Building, Publishing, and Previewing Documentation with the DocC-Plugin
 
 The preferred way of building documentation for your Swift package is by using
 the Swift-DocC Plugin. Refer to instructions in the plugin's 
```

Comment:
```suggestion
the Swift-DocC Plug-in. Refer to instructions in the plug-in's 
```

Does the first instance of "Plug-in" need to be initial-capped? If not, recommend making that lowercase.

Reply:
@chuckdude I would opt to retain the initial capitalization because it's the package's name